api/types/events/struct.go

```diff
@@ -83,7 +81,7 @@ func (s *Struct) MarshalJSON() ([]byte, error) {
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
-	return utils.FastMarshal(m)
+	return json.Marshal(m)
```

Comment:
Why was this changed from `utils.FastMarshal` to `json.Marshal`?

Reply:
This marshaling logic can't be moved out of `api` so I couldn't use `utils.FastMarshal` here. Is there a functional difference other than speed?

Reply:
No that's fine.